index.html

```diff
@@ -8,66 +8,40 @@
   <script class='remove'>
   var respecConfig = {
     group: "webperf",
-    shortName: "network-error-logging",

```

Comment:
Is shortName deprecated now? Or is it derived somehow?

Reply:
Sorry for the delay. It's derived from the "github" repo name, if shortName is missing.

Reply:
Cool, thanks! We should get the respec docs updated to mention that; [the `github` section](https://respec.org/docs/#github) talks about a few generated variables, but not`shortName`.